jenkins/automation/kubernetes/nightly-prvsn-sanity-deploy.groovy

```diff
@@ -0,0 +1,219 @@
+* You should have received a copy of the GNU Affero General Public License
+* along with this program. If not, see <https://www.gnu.org/licenses/>.
+* For any questions about this software or licensing,
+* please email opensource@seagate.com or cortx-questions@seagate.com.
+*/
+
+def remote_host() {
+    hosts = "${NODE_HOST_LIST}"
+    remotes = []
+    hosts = hosts.split("\n")
+    for (int num = 0; num < hosts.size(); num++) {
+        remote_node = [:]
+        remote_node.name = "node-" + num
+        remote_node.host = hosts[num].split(",")[0].split("=")[1]
+        remote_node.user = hosts[num].split(",")[1].split("=")[1]
+        remote_node.password = hosts[num].split(",")[2].split("=")[1]
+        remote_node.allowAnyHosts = true
+        remotes[num] = remote_node
+    }
+    return remotes
+}
+
+pipeline {
+    agent {
+        node {
+            label 'prvsnr_sanity_g4-rhev4-0658'
+        }
+    }
+
+    options {
+        timeout(time: 240, unit: 'MINUTES')
+        timestamps()
+        buildDiscarder(logRotator(daysToKeepStr: '20', numToKeepStr: '20'))
+    }
+
+    environment {
+        WORK_SPACE = "/root/cortx-k8s/k8_cortx_cloud"
+    }
+
+    parameters {
+
+        string(name: 'CORTX_SCRIPTS_REPO', defaultValue: 'https://github.com/Seagate/cortx-k8s', description: 'Repository for cortx-k8s scripts (Services Team)', trim: true)
+        string(name: 'CORTX_SCRIPTS_BRANCH', defaultValue: 'cortx-test', description: 'cortx-k8s scripts (Provisioner Team)', trim: true)
+        booleanParam(name: 'SETUP_K8s_CLUSTER', defaultValue: false, description: 'Selecting this option will setup K8s Cluster before running Deployment.')
+        choice (
+            choices: ['yes' , 'no'],
+            description: 'Selecting this option will run test sanity after Deployment ',
+            name: 'RUN_TEST_SANITY'
+        )
+    }
+
+    stages {
+        stage ("Define Variable") {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                script {
+                    env.allhost = sh( script: '''
+                    echo $NODE_HOST_LIST | tr ' ' '\n' | awk -F["="] '{print $2}'|cut -d',' -f1
+                    ''', returnStdout: true).trim()
+                
+                    env.master_node = sh( script: '''
+                    echo $NODE_HOST_LIST | tr ' ' '\n' | head -1 | awk -F["="] '{print $2}' | cut -d',' -f1
+                    ''', returnStdout: true).trim()
+
+                    env.hostpasswd = sh( script: '''
+                    echo $NODE_HOST_LIST | tr ' ' '\n' | head -1 | awk -F["="] '{print $4}'
+                    ''', returnStdout: true).trim()
+
+                }
+            }
+        }
+        stage('Destroy CORTX Cluster') {
+            steps {
+                script {
+                    remotes = remote_host()
+                    sshCommand remote: remotes[0], command: """
+                        cd ${WORK_SPACE}
```

Comment:
I guess we are in `WORKSPACE` directory everytime, so can you try running job by removing `cd ${WORKSPACE}` in each stage where it is used?